fix: updated the k8s documentation for NIM v2.19.0 release (#208)
nginx/documentation: content/nim/deploy/kubernetes/deploy-using-helm.md

If deploying on OpenShift, include this setting in the `values.yaml` file:

```yaml
openshift:
  enabled: true
```

### How OpenShift handles security constraints

When `openshift.enabled: true` is set in the `values.yaml` file, the NGINX Instance Manager deployment automatically creates a **custom Security Context Constraint (SCC)** and links it to the Service Account used by all pods.  

By default, OpenShift enforces strict security policies that require containers to run as **non-root** users. The NGINX Instance Manager deployment needs specific user IDs (UIDs) for certain services, such as **1000** for `nms` and **101** for `nginx` and `clickhouse`. Since the default SCCs do not allow these UIDs, a **custom SCC** is created. This ensures that the deployment can run with the necessary permissions while maintaining OpenShift’s security standards. The custom SCC allows these UIDs by setting the `runAsUser` field, which controls which users can run containers.  

> {{< note >}} If you’re encountering errors with the custom [Security Context Constraints](https://docs.openshift.com/container-platform/4.15/authentication/managing-security-context-constraints.html), it may be due to insufficient permissions to access the Security Context Constraints resource. Please contact a Cluster Administrator to request access, either via a ClusterRoleBinding or by adjusting your user role. {{< /note >}}

Reviewer: @balakoteswar I see this info in your link to Open Shift docs, and that's great. However: 

- The link also suggests that the info is moving. 
- I don't see `ClusterRoleBinding` either in our docs or in the Open Shift docs
   - I do see `clusterrolebinding` as an option for `kubectl` in various locations
   - In the Open Shift docs, I do see how a Cluster Administrator would update a "cluster role binding"
      - This info is the same after the move to the redhat.com domain. 

So I'm making the following change: 

```suggestion
{{< note >}} If you’re encountering errors with the custom [Security Context Constraints](https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html/authentication_and_authorization/managing-pod-security-policies), you may not have permissions to access the Security Context Constraints resource. Please contact a Cluster Administrator to request access, either through a cluster role binding or by adjusting your user role. {{< /note >}}
```

> To verify that the custom SCC has been created, run only after installing the helm chart:

Reviewer 2: ```suggestion
To verify that the custom SCC has been created, after installing the helm chart, run:
```

Author: Addressed.

   {{< important >}} Remember to save the password for future use. Only the encrypted password is stored, and there's no way to recover or reset it if lost. {{< /important >}}

> (Optional) Replace `<chart-version>` with the desired chart version. If omitted, the latest version will be installed. At present, only version 2.19.0 is supported.

Reviewer 2: ```suggestion
(Optional) Replace `<chart-version>` with the desired chart version. If omitted, the latest version will be installed. Currently only version 2.19.0 is supported.
```

Author: Addressed.

1. [Update the Helm repository list](#add-helm-repository).
1. [Adjust your `values.yaml` file](#create-a-helm-deployment-values.yaml-file) if needed.
1. To upgrade the NGINX Instance Manager deployment, run the following command. This command updates the `nms` deployment with a new version from the `nginx-stable/nms-hybrid` repository. It also hashes the provided password and uses the `values.yaml` file at the path you specify.

> (Optional) Replace `<chart-version>` with the desired chart version. If omitted, the latest version will be installed. At present, only version 2.19.0 is supported.

Reviewer 2: ```suggestion
(Optional) Replace `<chart-version>` with the desired chart version. If omitted, the latest version will be installed. Currently only version 2.19.0 is supported.
```

Author: Looks like it's repeated or am I missing anything here?
